cgmanifest.json

```diff
@@ -22058,6 +22058,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
```

Comment:
Add the new packages here

Reply:
I think you're still missing one for python-conda-libmamba-solver

Reply:
added